rskj-core/src/main/java/org/ethereum/rpc/parameters/HexStringParam.java

```diff
@@ -5,6 +5,10 @@
 
 public abstract class HexStringParam {
     HexStringParam(String hexString) {
+        if(hexString.isEmpty()) {
+            return;
```

Comment:
I left that behevior to the other classes that extends from this one, like `HexDurationParam`, which should not throw an error if the value is null since the methods using them behave different depending on that.